enhancements/console/custom-logos.md

```diff
@@ -0,0 +1,147 @@
+---
+title: custom-logos
+authors:
+  - "@cajieh"
+reviewers:
+  - "@jhadvig "
+  - "@spadgett"
+  - "@everettraven"
+  - "@JoelSpeed"
+approvers:
+  - "@spadgett"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2025-02-11
+last-updated: 2025-02-11
+---
+
+# Custom Logos
+
+## Summary
+
+The OpenShift Container Platform (OCP) web console is currently being upgraded to PatternFly 6. In PatternFly 6, the masthead color changes based on the mode (light or dark). Consequently, a single custom logo may not be suitable for both theme modes. 
+
+Add the ability to specify custom logos to support light and dark theme modes for the masthead and favicon.
+
+## Background info
+
+The OpenShift Container Platform (OCP) web console is currently being upgraded to PatternFly 6. In PatternFly 6, the masthead color changes based on the mode (light or dark). Consequently, a single custom logo may not be suitable for both modes.
+
+To address this, we need to allow users to add custom logos for both the masthead and favicon that are compatible with light and dark themes. This ensures that the logos are always visible and aesthetically pleasing, regardless of the theme mode.
+
+The custom logos feature will enable users to specify different logos for the masthead and favicon based on the theme mode. This will involve exposing a new API endpoint to support custom logos for both light and dark themes.
+
+## Motivation
+
+The OpenShift Container Platform (OCP) web console is currently being upgraded to PatternFly 6. In PatternFly 6, the masthead color changes based on the mode (light or dark). Consequently, a single custom logo may not be suitable for both theme modes. This is evident with the existing OKD and OpenShift logos, which assume a dark masthead background and include white text.
+
+### User Stories
+
+* As an OpenShift web console user, I want to be able to add custom logos for light and dark theme modes in the masthead and favicon.
+
+### Goals
+
+This feature should allow users to add custom logos for the masthead and favicon that are compatible with both light and dark theme modes in the OpenShift web console.
+
+### Non-Goals
+
+
+## Proposal
+
+### Workflow Description
+
+├── spec
+│   ├── customization
+│       ├── customLogos
+│           ├── type
+│           ├── themes
+│               ├── type
+│               ├── file
+│                   ├── key
+│                   ├── name
+└── ...
+
+### Risks and Mitigations
+
+1. Users could set both the `CustomLogoFile` and `CustomLogos` APIs. The `CustomLogos` API configuration will take precedence over the old `CustomLogoFile` field.
+
+2. If only the dark or light theme is specified, the default OCP logo will be used for the unspecified theme.
```

Comment:
I would rephrase this to:
Each of the Console supported themes can be configured individually by setting either the `Light` or `Dark` theme type or by applying a default theme to all supported themes using the `Default` theme type. If the `Default` theme type is set along with a specific `Dark` or `Light` theme, the specific theme setting will override the default one.